client/my-sites/plans-features-main/index.tsx

```diff
@@ -962,7 +962,12 @@ const PlansFeaturesMain = ( {
 												onStorageAddOnClick={ handleStorageAddOnClick }
 												showRefundPeriod={ isAnyHostingFlow( flowName ) }
 												planTypeSelectorProps={
-													! hidePlanSelector ? planTypeSelectorProps : undefined
+													! hidePlanSelector
+														? {
+																...planTypeSelectorProps,
+																onPlanIntervalChange: handlePlanIntervalChange,
```

Comment:
Can't we add this to `planTypeSelectorProps` above and avoid updating the props on every usage?